src/main/test/ApplicationUtilsTests.cpp

```diff
@@ -488,146 +375,14 @@ TEST_CASE("offline self-check works", "[applicationutils][selfcheck]")
         damage.damageVictim();
         REQUIRE(selfCheck(chkConfig) == 1);
     }
-    {
-        // Damage the SQL ledger.
-        TemporarySQLiteDBDamager damage(chkConfig);
-        damage.damageVictim();
-        REQUIRE(selfCheck(chkConfig) == 1);
-    }
 }
 
 TEST_CASE("application setup", "[applicationutils]")
 {
     VirtualClock clock;
-
-    SECTION("SQL DB mode")
```

Comment:
you could also keep this on PG, right? (not that I'm insisting, just trying to understand)

Reply:
This test is currently pretty meaningless for PG, given that the bulk of this test is just a call to `BucketListIsConsistentWithDatabase`, and that invariant now only checks for offers because it assumes BucketListDB is enabled. This won't break anything, but you're not gonna get better coverage than what we already have.